packages/components/app/styles/components/application-state.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// APPLICATION-STATE COMPONENT
+//
+$hds-application-state-padding: 12px 0;
+
+.hds-application-state {
+  align-self: center;
+  width: 20rem;
+  max-width: 100%;
+  margin: 0 auto;
+}
+
+.hds-application-state__header,
+.hds-application-state__body,
+.hds-application-state__footer {
+  display: flex;
+}
+
+.hds-application-state__header {
+  display: grid;
+  grid-row-gap: 0;
+  grid-column-gap: 0;
```

Comment:
The default gap size is "0" so I probably wouldn't include these gap styles unless you are worried that it may somehow inherit values from somewhere it shouldn't.

Reply:
Although you _could_ add a value for grid-column-gap and then size "hds-application-state__header-icon" to precisely fit the nested SVG. (That way you could use gap for spacing between the icon and text instead of making the icon container wider than the SVG.)

Reply:
for all reviewers: I definitely am not 100% set on the CSS approach, I'm fine to adapt to what @KristinLBradley has suggested, or can change to something else if it makes more sense.